com.icons8.Lunacy.yml

```
@@ -0,0 +1,63 @@
+app-id: com.icons8.Lunacy
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: lunacy
+build-options:
+  no-debuginfo: true
+tags:
+  - proprietary
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=cups
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

If this actually supports Wayland.

---

I don't think it supports Wayland. When I tried Wayland + Fallback-X11 using Flatseal, it didn't launch.